python/cudf/cudf/tests/test_concat.py

```diff
@@ -1921,3 +1921,21 @@ def test_concat_mixed_list_types_error(s1, s2):
 
     with pytest.raises(NotImplementedError):
         cudf.concat([s1, s2], ignore_index=True)
+
+
+def test_horizontal_concat_dictionary():
+
+    d = {
+        'first': cudf.DataFrame({'A': [1, 2], 'B': [3, 4]}),
+        'second': cudf.DataFrame({'A': [5, 6], 'B': [7, 8]}),
+    }
```

Comment:
Can we parametrize to include a few additional test cases?

- A single entry in `d`
- A third entry in `d` with just 1 key-value pair

Reply:
Aren't single entry in `d` and a third entry in `d` with just 1 key-value pair the same thing? Are you saying there should be a test for `d = {'first': cudf.DataFrame({'A': [1, 2], 'B': [3, 4]})}`?